### PR TITLE
Add a couple of missing getters

### PR DESCRIPTION
**Pull request by uajain**
Adds getters for "no-pull" and "no-deploy".

Makes work in gnome-software easier to identify a regular update or is a autoupdate that needs to be run with "no-deploy" and then "no-pull".

### CODE REVIEW

File: common/flatpak-transaction.c [hunk 1]
```diff
@@ -1196,6 +1196,23 @@ flatpak_transaction_set_no_pull (FlatpakTransaction *self,
...
 }
 
+/**
+ * flatpak_transaction_get_no_pull:
+ * @self: a #FlatpakTransaction
+ *
+ * Gets whether the transaction should operate only on locally
+ * available data.
+ *
+ * Returns: %TRUE if no_pull is set, %FALSE otherwise
```
Comment (pwithnall) [inline common/flatpak-transaction.c]: Missing `Since:` (same below)

File: common/flatpak-transaction.c [hunk 2]
```diff
@@ -1213,6 +1230,23 @@ flatpak_transaction_set_no_deploy (FlatpakTransaction *self,
   priv->no_deploy = no_deploy;
 }
 
+/**
+ * flatpak_transaction_get_no_deploy:
```
Comment (pwithnall) [inline common/flatpak-transaction.c]: Symbols missing from `doc/reference/flatpak-sections.txt` (same above)

**Comment by pwithnall** [comment]
Could also with some unit tests, which should be trivial to add

File: common/flatpak-transaction.c (hunk 1 above)
Comment (uajain) [reply common/flatpak-transaction.c]: Addressed.

File: common/flatpak-transaction.c (hunk 2 above)
Comment (uajain) [reply common/flatpak-transaction.c]: Addressed.

**Comment by uajain** [comment]
> Could also with some unit tests, which should be trivial to add

Addressed in the latest iteration

**Pull request merged**